frontend/src/App.js

```diff
@@ -1,11 +1,9 @@
 import React from 'react';
 import Main from './Main';
-import Header from './Header';
-
+import TopBar from './TopBar'
 function App() {
   return (
     <>
-      <Header />
       <Main />
     </>
   );
```

Comment:
I mentioned this in another comment, we should be adding elements that exist on every page (i.e. the header/navigation bar) to this component instead of to each page individually.